e2e/scenarios/basic/res/spidsamlcheck-conf/idp.json

```
@@ -0,0 +1,27 @@
+{
+	"enabled": true,
+	"basepath": "/",
+	"entityID": "https://spid-saml-check.ecs.dev.pn.pagopa.it",
```

```suggestion
	"entityID": "http://spid-saml-check:8088",
```

Even though this line has no practical effect in the E2E test case, I would keep the validato configuration aligned as well, just in case future scenarios might require also validator functionalities